src/index.ts

```diff
@@ -39,13 +65,22 @@ const endpoints = [
 const onEvent =
   (pageview = false) =>
   async (event: MCEvent) => {
+    const eventId = pageview ? crypto.randomUUID() : null
+    const hashedUserData = await hashUserData(event)
     for (const { url, data } of endpoints) {
       const payload = {
         ...getStandardParams(event),
         ...data,
         ...event.payload,
+        ...hashedUserData,
+      }
+
+      if (pageview) {
+        payload.events = '[["pageview", null]]'
+        if (!payload.event_id) {
+          payload.event_id = eventId
+        }
```

Comment:
```suggestion
          payload.event_id = payload.event_id || crypto.randomUUID()
```

Reply:
In case of a pageview, we need to send the same event_id to two different endpoints. So I cannot generate UUID here, since if I do that it will be sending two different ones. If we want to get rid of the if statement, we can do:
`payload.event_id =  payload.event_id || eventId`. Is this better?